src/layout/Footer/Footer.tsx

```diff
@@ -1,9 +1,78 @@
-import { FooterWrapper } from './Footer.styled';
+import { AppBar, IconButton, Menu, MenuItem, Typography } from '@mui/material';
+import { StyledToolBar } from './Footer.styled';
+import LanguageRoundedIcon from '@mui/icons-material/LanguageRounded';
+import { useState } from 'react';
+
+const languages: string[] = ['English/en', 'Espanol/es'];
 
 export const Footer = () => {
+    const [selectedLanguage, setSelectedLanguage] = useState('English/en');
+    const [anchorElUser, setAnchorElUser] = useState<null | HTMLElement>(null);
+
+    const handleOpenLanguageMenu = (event: React.MouseEvent<HTMLElement>) => {
+        setAnchorElUser(event.currentTarget);
+    };
+
+    const handleCloseLanguageMenu = (setting: string) => {
+        setAnchorElUser(null);
+        if (languages.includes(setting)) {
+            setSelectedLanguage(setting);
+        }
+    };
     return (
-        <FooterWrapper>
-            <h2>Footer</h2>
-        </FooterWrapper>
+        <>
+            <AppBar position="static" color="primary">
+                <StyledToolBar>
+                    <Typography variant="caption" color="inherit">
+                        © 2024 All rights reserved
+                    </Typography>
+
+                    <IconButton
```

Comment:
Change icon button to regular button so that the oval hover effect doesn't happen.